munimap/queue/worker.py

```diff
@@ -19,6 +19,9 @@
 
     file_config = Config(root_path='./')
 
+    formatter = logging.Formatter('%(asctime)s %(levelname)s: %(message)s ')
```

Comment:
isn't this something that should be configured outside of the actual code?

Reply:
maybe true, yes. This could be done via the config file. I think if the need to change this arises, we can still make this configurable.